Bottom: 4ea33494b58af67656b748acc5cc12b9b0d70269
Top:    3d0889913b08a2941c4755678d020b4a68c8c5aa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 00:50:56 +0100

Refresh of add_pours

---

diff --git a/src/draw.c b/src/draw.c
index ff27c6a..0992b55 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -947,7 +947,7 @@ DrawLayerGroup (int group, const BoxType * screen)
 	    {
 	      info.Layer = Layer;
 	      info.arg = True;
-        info.clip = screen;
+	      info.clip = screen;
 	      r_search (Layer->pour_tree, screen, NULL, pour_callback, &info);
 	      info.arg = False;
